### ISSUE TITLE
Feature request: Drag-and-drop ordering of items

### ISSUE THREAD

**Issue by muhuk** [rcv4/blenderTodo#1]
It would be nice to be able to order todo items by dragging.

**Comment by rcv4** [comment]
Agree! Sadly its not yet supported by blender, but there is a Task for it already:
https://developer.blender.org/T67893
